### DISCUSSION TITLE
Im brent

### DISCUSSION THREAD

**Discussion by Brentkc-sys** [open-policy-agent/community#636] [Gatekeeper]
Hi im brent, im the gatekeeper and need some help

**Comment by charlieegan3** [comment]
Hey Brent, you might want to check out the gatekeeper channel in the OPA slack: https://openpolicyagent.slack.com/archives/CDTN970AX

**Reply by charlieegan3** [reply]
Signup link: http://slack.openpolicyagent.org

**Reply by Brentkc-sys** [reply]
I seriously don't know anything about this website and need a few pointers on getting started can you help

**Reply by anderseknert** [reply]
Hi there 👋 If you want to know more about OPA, the documentation should be where to start: https://www.openpolicyagent.org/docs/latest/
